v23.1/user-defined-functions.md

```diff
@@ -12,12 +12,15 @@ A user-defined function (UDF) is a named function defined at the database level
 
 The basic components of a user-defined function are a name, list of arguments, return type, volatility, language, and function body.
 
-- An argument has a _mode_ and a _type_. CockroachDB supports the `IN` argument mode. The type can be a built-in type, [user-defined enum](enum.html), or implicit record type. CockroachDB **does not** support default values for arguments.
-- The return type can be a built-in type, user-defined enum, implicit record type, or `VOID`. `VOID` indicates that there is no return type and `NULL` will always be returned. If the return type of the function is not `VOID`, the last statement of a UDF must be a `SELECT`.
+- An argument has a _mode_ and a _type_. CockroachDB supports the `IN` argument mode. The type can be a built-in type, [user-defined `ENUM`](enum.html), or implicit record type. CockroachDB **does not** support default values for arguments.
+- The return type can be a built-in [type](data-types.html), user-defined [`ENUM`](enum.html), [`RECORD`](create-function.html#create-a-function-that-returns-a-record-type), implicit record type, or `VOID`.
+    - Preceding a type with `SETOF` indicates that a set, or multiple rows, should be returned. For an example, see [Create a function that returns a set of results](create-function.html#create-a-function-that-returns-a-set-of-results).
+    - `VOID` indicates that there is no return type and `NULL` will always be returned. If the return type of the function is not `VOID`, the last statement of a UDF must be a `SELECT`.
```

Comment:
Currently only `Select` is supported in UDFs, so each statement in a UDF must be a `SELECT` in any event.

Reply:
Noted - commenting the last line out until relevant.